src/Uno.UI/Microsoft/UI/Xaml/Media/RadialGradientBrush.Android.cs

```diff
@@ -5,12 +5,21 @@
 using Windows.UI.Xaml.Media;
 using Point = Windows.Foundation.Point;
 using Size = Windows.Foundation.Size;
+using Rect = Windows.Foundation.Rect;
 
 namespace Microsoft.UI.Xaml.Media
 {
 	partial class RadialGradientBrush
 	{
-		protected internal override Shader GetShader(Size size)
+		protected override Paint GetPaintInner(Rect destinationRect)
+		{
+			var paint = new Paint();
+			paint.SetShader(GetShader(destinationRect.Size));
```

Comment:
Should we have Antialias applied here?

Reply:
That is a copy from https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Media/GradientBrush.Android.cs

I think if we want, we should do it in a single place, not in every GetPaintInner override.